x-pack/legacy/plugins/infra/common/http_api/log_analysis/index_patterns/validate.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as rt from 'io-ts';
+
+export const LOG_ANALYSIS_INDEX_PATTERNS_VALIDATE_PATH =
+  '/api/infra/log_analysis/index_patterns/validate';
```

Comment:
My reasoning was to group all actions (in this case only `validate`) that can be done with an entity (`indices`). For the categorization we can have something like 

```
/api/infra/log_analysis/categorization/some_action
/api/infra/log_analysis/categorization/validate
...
```

I lean more towards this approach because it keeps all code related to a certain entity together, but if we are doing it the other way around in kibana I'm fine changing it.

Reply:
That's a possibility too. My suggestion is based on the fact that the inverse hierarchy has been used for the log entry rate results (`log_analysis/results/log_entry_rate`). I can see an argument for either, but we should not mix them. If you want to go with `entity/{verb,attribute}`, then we should change the log entry rate URL and directory structures to match.

Reply:
I'll switch it to match what we have already then. If we feel it the other way around is better we can always change it later :)